flexget/plugins/input/json.py

```diff
@@ -0,0 +1,137 @@
+"""Plugin for json file."""
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # noqa pylint: disable=unused-import, redefined-builtin
+import dateutil.parser as parser
+
+import re
+import os
+import logging
+import glob
+
+from flexget import plugin
+from flexget.utils import json
+from flexget.entry import Entry
+from flexget.event import event
+
+log = logging.getLogger('json')
+
+
+class Json(object):
+    """
+    Parse a json file for entries using regular expression.
+
+    :: 
+
+      files: <path to JSON file(s)>
+      encoding: <JSON encoding>
+      entry:
+        - <field>: <corresponding JSON key>
+        - <field>
+        - get_remnants
+
+    Note: each entry must have at least two fields, 'title' and 'url'. If not specified in the config, 
+    this plugin asssumes that keys named 'title' and 'url' exist within the JSON.
+
+    Example::
+
+      json:
+        files: entries.json
+        encoding: utf8
+        entry:
+          - title: name
+          - url: 'web_.*'
+          - get_remnants
+          
+    """
+    
+    schema = {
+        'type': 'object',
+        'properties': {
+            'files': {'type': 'string'},
```

Comment:
if you set `'format': 'path'` it will enforce file actually existing. Also calling if `files` and not having it be multiple files is kinda weird

Reply:
I can look into this. The way it is setup now, the user can specify either a json file: `/home/username/json_files/my_file.json` or a directory containing all of the json files by using: `/home/user/json_files/*.json`